### PR TITLE
Specify data collected in HTTP client errors

### PR DESCRIPTION
**Pull request by martinhaintz**
## DESCRIBE YOUR PR
Add additional informations what will be sent to sentry in the dart http integration.
closes https://github.com/getsentry/sentry-dart/issues/2130

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/dart/integrations/http-integration.mdx [hunk 1]
```diff
@@ -21,6 +21,8 @@ var uriResponse = await client.post('https://example.com/whatsit/create',
 }
 ```
 
+The error event sent to sentry contains the "request method", "request url", "request header", "request body", "request content-length", "request duration", "response headers", "response content-length" and the "response status code".
```
Comment (buenaflor) [inline docs/platforms/dart/integrations/http-integration.mdx:24]:
```md
When an error occurs, the following information is captured and sent to Sentry:

Request details:
- Method
- URL
- Headers
- Body
- Content length
- Duration

Response details:
- Headers
- Content length
- Status code
```

what do you think of this

Comment (martinhaintz) [reply docs/platforms/dart/integrations/http-integration.mdx]: I like it and changed it.

**Review by buenaflor — APPROVED** [review]

File: docs/platforms/dart/integrations/http-integration.mdx [hunk 2]
```diff
@@ -21,6 +21,23 @@ var uriResponse = await client.post('https://example.com/whatsit/create',
...
+- Headers
+- Body
+- Content length
+- Duration
+
+Response details:
+
+- Headers
+- Content length
+- Status code
```
Comment (kahest) [inline docs/platforms/dart/integrations/http-integration.mdx:39]: `m`: can we indicate which of these are subject to scrubbing of sensitive data per default? and maybe add a link to https://docs.sentry.io/platforms/flutter/data-management/sensitive-data/

**Review by vivianyentran — APPROVED** [review]
+1 to @kahest 's suggestion

**Comment by buenaflor** [comment]
Can we get a quick rerun of the vercel workflows pls

**Comment by kahest** [comment]
@buenaflor @martinhaintz vercel rerun ✅

**Pull request merged**